hikvision-nvr/info.json

```diff
@@ -0,0 +1,515 @@
+{
+  "name": "hikvision-nvr",
+  "version": "1.0.0",
+  "label": "Hikvision NVR",
+  "description": "Hikvision's Network Video Recorders (NVRs) provide advanced artificial intelligence capabilities for any connected data stream, even those from conventional security cameras",
+  "publisher": "Fortinet CSE",
```

Comment:
I think publisher should be Fortinet instead of Fortinet CSE

Reply:
This connector was submitted by the Fortinet CSE team, for this reason publisher is Fortinet CSE.